src/NABfunctions.ts

```diff
@@ -127,7 +127,9 @@ export async function FindTranslatedTexts() {
             if (textToSearchFor === '') {
                 throw new Error('This line does not contain any translated property or label.');
             }
-            await VSCodeFunctions.FindTextInFiles(textToSearchFor, false, '*.xlf');
+            let fileFilter = '';
+            if (Settings.GetConfigSettings()[Setting.SearchOnlyXlfFiles] === true) { fileFilter = '*.xlf'; }
+            await VSCodeFunctions.FindTextInFiles(textToSearchFor, false, fileFilter);
```

Comment:
Since `VSCodeFunctions.FindTextInFiles` is only called from these to functions the setting check could be performed inside `FindInTextFiles`. Or perhaps by creating a wrapping function like `FindInXlfFiles(...)` which could call `FindInTextFiles` with or without the filter?